Directory.Packages.props

```diff
@@ -1,234 +1,152 @@
-    <PackageVersion Include="Microsoft.AspNetCore.Components.WebAssembly.DevServer" Version="7.0.20" />
-    <PackageVersion Include="Microsoft.AspNetCore.Components.WebAssembly.Server" Version="7.0.20" />
-    <PackageVersion Include="Microsoft.AspNetCore.DataProtection.Abstractions" Version="7.0.20" />
-    <PackageVersion Include="Microsoft.Data.Sqlite" Version="7.0.20" />
-    <PackageVersion Include="Microsoft.Data.Sqlite.Core" Version="7.0.20" />
-    <PackageVersion Include="Microsoft.EntityFrameworkCore" Version="7.0.20" />
-    <PackageVersion Include="Microsoft.EntityFrameworkCore.Design" Version="7.0.20" />
-    <PackageVersion Include="Microsoft.EntityFrameworkCore.Relational" Version="7.0.20" />
-    <PackageVersion Include="Microsoft.EntityFrameworkCore.Sqlite" Version="7.0.20" />
-    <PackageVersion Include="Microsoft.EntityFrameworkCore.SqlServer" Version="7.0.20" />
-    <PackageVersion Include="Microsoft.Extensions.Caching.Abstractions" Version="7.0.0" />
-    <PackageVersion Include="Microsoft.Extensions.Caching.Memory" Version="7.0.0" />
-    <PackageVersion Include="Microsoft.Extensions.Configuration" Version="7.0.0" />
-    <PackageVersion Include="Microsoft.Extensions.Configuration.Abstractions" Version="8.0.0" />
-    <PackageVersion Include="Microsoft.Extensions.Configuration.Json" Version="7.0.0" />
-    <PackageVersion Include="Microsoft.Extensions.DependencyInjection" Version="8.0.1" />
-    <PackageVersion Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="8.0.2" />
-    <PackageVersion Include="Microsoft.Extensions.Hosting.Abstractions" Version="7.0.0" />
-    <PackageVersion Include="Microsoft.Extensions.Http" Version="7.0.0" />
-    <PackageVersion Include="Microsoft.Extensions.Http.Polly" Version="7.0.20" />
-    <PackageVersion Include="Microsoft.Extensions.Logging" Version="8.0.0" />
-    <PackageVersion Include="Microsoft.Extensions.Logging.Abstractions" Version="8.0.1" />
-    <PackageVersion Include="Microsoft.Extensions.Logging.Console" Version="7.0.0" />
-    <PackageVersion Include="Microsoft.Extensions.Options" Version="8.0.0" />
-    <PackageVersion Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="7.0.0" />
-    <PackageVersion Include="Npgsql" Version="7.0.8" />
-    <PackageVersion Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="7.0.18" />
-    <PackageVersion Include="Oracle.EntityFrameworkCore" Version="7.21.13" />
-    <PackageVersion Include="Polly" Version="7.2.4" />
-    <PackageVersion Include="Pomelo.EntityFrameworkCore.MySql" Version="7.0.0" />
-    <PackageVersion Include="Refit" Version="7.2.22" />
-    <PackageVersion Include="Refit.HttpClientFactory" Version="7.0.0" />
-    <PackageVersion Include="System.Formats.Asn1" Version="8.0.1" />
-    <PackageVersion Include="System.Text.Json" Version="8.0.5" />
-  </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)' == 'net8.0'">
```

Comment:
Hello @sfmskywalker Removing this causing problem in .Net8.0 Target framework packages

Reply:
That's odd - the Elsa.Server.Web project seems to build fine when targeting .NET 8.0.

Do you have a repro?

Reply:
I think as Target Framework 6.0 is removed, particular  to those package section can be removed.
For TargetFramework 8.0 and 9.0, can we still go with validation Condition check as below 
<ItemGroup Condition="'$(TargetFramework)' == 'net8.0'"><PackageVersions>.

Reply:
We could, but it’s better if we don’t have to, keeps things simple. I haven’t seen any issues so far with referencing 9.0 packages. Can you show me the issue?